src/createServer.js

```diff
@@ -1,11 +1,37 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { userRouter } = require('./routes/users.route');
+const { expensesRouter } = require('./routes/expenses.route');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  /* CORS  */
+  const options = {
+    origin: '*',
+    methods: 'GET, POST, PUT, DELETE , PATCH',
+    allowedHeaders: 'Content-Type',
+    credentials: true,
+  };
+
+  app.use(cors(options), express.json());
+
+  /* ROUTES */
+  app.use('/users', userRouter);
+  app.use('/expenses', expensesRouter);
+
+  /* ERRORS */
+  app.use((_, res) => {
+    res.status(404).json({ message: 'Not found' });
+  });
+
+  app.use((err, _, res) => {
+    res.status(500).json({ message: err.message });
+  });
```

Comment:
The error handler should include the `next` parameter to ensure it matches the signature for Express error-handling middleware, even if it's not used in this case.